### PR TITLE
Revisit Memberlist defaults

### PR DESCRIPTION
**Pull request by joe-elliott**
**What this PR does**:
The latest Cortex upgrade started correctly applying Memberlist LAN defaults. This caused a spike in resource usage by memberlist ring propagation. This PR looks to choose some sensible defaults that balances resource usage against ring propagation speed. With the proposed defaults in a 125 node memberlist cluster and a ring heartbeat of 5s:

CPU Usage: .04
Bandwidth (tx + rx): 346 KBs
Average heartbeat age: 15s

**Which issue(s) this PR fixes**:
Fixes #<issue number>

**Checklist**
- [ ] Tests updated
- [x] Documentation added
- [ ] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

File: docs/tempo/website/configuration/manifest.md [hunk 1]
```diff
@@ -273,14 +280,14 @@ ingester:
...
     port: 0
-    id: Annanays-Mac.local
+    id: hostname
   concurrent_flushes: 16
   flush_check_period: 30s
   flush_op_timeout: 5m0s
   trace_idle_period: 30s
   max_block_duration: 1h0m0s
   max_block_bytes: 1073741824
-  complete_block_timeout: 6m0s
```
Comment (kvrhdn) [inline docs/tempo/website/configuration/manifest.md:283]: I'm curious why some of these changed, have we not updated this file for a while?

**Review by kvrhdn — APPROVED** [review]
LGTM, nice work!

File: docs/tempo/website/configuration/manifest.md (hunk 1 above)
Comment (joe-elliott) [reply docs/tempo/website/configuration/manifest.md:283]: yes, a lot of these changes are just b/c we haven't updated in a bit. that one in particular was my miss in the tenant index PR.

honestly this file should be auto generated in CI :P

Comment (kvrhdn) [reply docs/tempo/website/configuration/manifest.md:283]: https://github.com/grafana/tempo/issues/890

**Pull request merged**